src/features/calendar/components/index.tsx

```diff
@@ -55,17 +55,19 @@ const Calendar = () => {
   );
 
   useEffect(() => {
-    setFocusDate(getDateFromString(focusDateStr));
-  }, [focusDateStr]);
-
-  useEffect(() => {
-    setSelectedTimeScale(getTimeScale(timeScaleStr));
-  }, [timeScaleStr]);
-
-  useEffect(() => {
-    router.query.focusDate = dayjs(focusDate).format('YYYY-MM-DD');
-    router.query.timeScale = selectedTimeScale;
-    router.push(router, undefined, { shallow: true });
+    const focusedDate = dayjs(focusDate).format('YYYY-MM-DD');
+    router.push(
+      {
+        pathname: undefined,
+        query: {
+          focusDate: focusedDate,
+          orgId: orgId,
```

Comment:
You are missing the `campId` here, which is necessary when using this calendar on the campaign ("project") page. Currently, this error is thrown (note the URL to reproduce):

<img width="1211" alt="image" src="https://github.com/zetkin/app.zetkin.org/assets/550212/0b1ae6f0-ee5c-4bc9-b6a7-660f80f1ab41">